examples/viewer_lib/ui/segmentation/islands_effect_ui.py

```diff
@@ -2,14 +2,16 @@
 from enum import Enum, auto
 
 from trame_server.utils.typed_state import TypedState
-from trame_vuetify.widgets.vuetify3 import VBtn, VContainer, VNumberInput, VSelect
+from trame_vuetify.widgets.vuetify3 import VBtn, VBtnToggle, VNumberInput, VSpacer
 from undo_stack import Signal
 
+from ..utils import FlexContainer
+
 
 class IslandsSegmentationMode(Enum):
     KEEP_LARGEST_ISLAND = auto()
     REMOVE_SMALL_ISLANDS = auto()
-    SPLIT_TO_SEGMENTS = auto()
+    SPLIT_TO_SEGMENT = auto()
```

Comment:
Rename to split to segments

This effect will generate one or more segments so plural makes sense (and is consistent with 3D Slicer naming)